util/avatars.ts

```diff
@@ -0,0 +1,15 @@
+export const githubProfileAvatarUrl = (username: string): string =>
+  `https://avatars.githubusercontent.com/${username}`;
+
+export const getAcronymFromString = (str: string): string =>
```

Comment:
This shouldn't be on a file called avatars, as this can be used for any sort of string.

Reply:
If you are talking about file name. you thinks we should have a file call `string` and a file `github` ?

Reply:
I think this specific one (the lower one) should be something like `stringUtils.ts` and the upper one (`githubUtils.ts`) or maybe just `strings.ts and `github.ts`